src/main.js

```diff
@@ -23,12 +25,24 @@ Vue.use(VueGoogleMaps, {
   },
 });
 
+Vue.config.lang = 'en_US';
+
+const messages = {
+  en_US: require('cp-translations/strings/en_US/messages.po'),
+};
+
+const i18n = new VueI18n({
+  locale: 'en_US',
+  messages,
```

Comment:
Why not set the default here rather than in the selector's directive ?
https://kazupon.github.io/vue-i18n/en/fallback.html